test/dynamo/test_autograd_function.py

```diff
@@ -546,14 +546,14 @@ def forward(self, L_x_: "f32[]", L_z_: "f32[]", L_weird_b: "f32[]", L_weird_c: "
         return (autograd_function_apply,)
 
     class fwd_body_0(torch.nn.Module):
-        def forward(self, ctx, x: "f32[]", z: "f32[]", l_weird_b: "f32[]", l_weird_c: "f32[]"):
+        def forward(self, ctx : torch.autograd.function.Function, x: "f32[]", z: "f32[]", l_weird_b: "f32[]", l_weird_c: "f32[]"):
```

Comment:
The context is a torch.autograd.funciton.Function instead of https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/side_effects.py#L270

Reply:
What causes some of these to state their types but some of them not to?